crossing-training.lic

```diff
@@ -403,13 +403,22 @@ class CrossingTraining
   end
 
   def research
+    if @settings.cyclic_research
```

Comment:
You'll want to add a default value for this setting to base.yaml.

I'd appreciate seeing an example of this in a character's YAML, but it is not required.